modules/nw-sriov-cfg-bond-interface-with-virtual-functions.adoc

```diff
@@ -0,0 +1,113 @@
+// Module included in the following assemblies:
```

Comment:
@tmulquee This file to be feature content. If it's feature content, it needs to go in a separate PR that you open against `main`.
CC @stevsmit

Reply:
I agree. Seems like it's more suitable for a separate PR given the PR title.

Reply:
Thanks, it's a feature for which I'm doing the RN and docs. I will close this PR and create a fresh with with RN content only.